skyportal/plot.py

```diff
@@ -793,6 +796,12 @@ def spectroscopy_plot(obj_id, user, spec_id=None, width=600, height=300):
     plot.xaxis.axis_label = 'Wavelength (Å)'
     plot.yaxis.axis_label = 'Flux'
     plot.toolbar.logo = None
+    if obj.redshift is not None and obj.redshift > 0:
+        plot.extra_x_ranges = {"rest_wave": Range1d(start=xmin_rest, end=xmax_rest)}
+        plot.add_layout(
+            LinearAxis(x_range_name="rest_wave", axis_label="Rest Wavelength (Å)"),
```

Comment:
Will "Å" always render for folks coded like this?

Reply:
Good question.  This was a cut and paste from the other axis label.  Since that other axis has been used for quite a while without issue, I would assume it will.